### PR TITLE
fix: add and verified projectID in audience

### PR DESCRIPTION
**Pull request by fgerschwiler**

### CODE REVIEW

File: internal/auth/repository/eventsourcing/eventstore/auth_request.go [hunk 1]
```diff
@@ -109,6 +109,8 @@ func (repo *AuthRequestRepo) CreateAuthRequest(ctx context.Context, request *mod
 		return nil, err
 	}
 	request.Audience = appIDs
+	projectIDAud := request.GetScopeProjectIDsForAud()
```
Comment (livio-a) [inline internal/auth/repository/eventsourcing/eventstore/auth_request.go:112]: didn't we decide to append the projectID as scope if none is provided as well?

File: internal/auth_request/model/request.go [hunk 2]
```diff
@@ -19,7 +19,8 @@ const (
 )
 
 const (
-	OrgDomainPrimaryScope = "urn:zitadel:org:domain:primary:"
+	OrgDomainPrimaryScope = "urn:zitadel:iam:org:domain:primary:"
+	ProjectIDAudScope     = "urn:zitadel:iam:org:project:id:aud:"
```
Comment (livio-a) [inline internal/auth_request/model/request.go:23]: @fforootd didn't we state to go with: `urn:zitadel:iam:org:project:{projectid}:aud`

**Review by livio-a — CHANGES_REQUESTED** [review]

File: site/docs/documentation/03-openidoauth.en.md [hunk 3]
```diff
@@ -67,6 +67,7 @@ In addition to the standard compliant scopes we utilize the following scopes.
...
 | urn:zitadel:iam:org:domain:primary:{domainname} | When requesting this scope **ZITADEL** will enforce that the user is a member of the selected organisation. If the organisation does not exist a failure is displayed | urn:zitadel:iam:org:domain:primary:acme.ch |
 | urn:zitadel:iam:role:{rolename}                 |                                                                                                                                                                 |                                            |
+| urn:zitadel:iam:org:project:id:aud:{projectid}  | By adding this scope, the requested projectid will be added to the audience of the access and id token                                                          | Zitadel Project: urn:zitadel:iam:org:project:id:aud:69234237810729019                                           |
```
Comment (fforootd) [inline site/docs/documentation/03-openidoauth.en.md:70]:
```suggestion
| urn:zitadel:iam:org:project:id:aud:{projectid}  | By adding this scope, the requested projectid will be added to the audience of the access and id token                                                          | ZITADEL Project: urn:zitadel:iam:org:project:id:aud:69234237810729019                                           |
```

Comment (fforootd) [reply site/docs/documentation/03-openidoauth.en.md:70]: Is it not ...:id:04913123123123:aud ?

**Review by livio-a — APPROVED** [review]

**Review by livio-a — APPROVED** [review]

**Pull request merged**